user_tools/src/spark_rapids_tools/tools/model_xgboost.py

```diff
@@ -551,8 +551,21 @@ def scan_tbl(
     sqls_to_drop = set()
 
     # Load job+stage level agg metrics:
-    job_stage_agg_tbl = scan_tbl('job_+_stage_level_aggregated_task_metrics')
-    if not any([job_stage_agg_tbl.empty, job_map_tbl.empty]):
+    job_agg_tbl = scan_tbl('job_level_aggregated_task_metrics')
+    stage_agg_tbl = scan_tbl('stage_level_aggregated_task_metrics')
+
+    if not any([job_agg_tbl.empty, stage_agg_tbl.empty, job_map_tbl.empty]):
```

Comment:
Might need to create an empty `job_stage_agg_tbl` for L712 if this conditional ever fails (not sure what might break if it's `None` vs. empty).

Reply:
Thanks @leewyang. Since this has been merged in a staging branch, updated the staging->dev PR with this change (#1050):

Commit Link: https://github.com/NVIDIA/spark-rapids-tools/pull/1050/commits/cbeaaff8f43c3073d779fbfd6c2911df5ecb9384